pkg/util/helm/helm.go

```diff
@@ -91,7 +91,7 @@ func (c cmd) Uninstall(t test.TestHelper) {
 	if c.namespace == "" {
 		t.Fatalf("namespace must be specified to uninstall helm release")
 	}
-	shell.Executef(t, "helm uninstall %s -n %s", c.release, c.namespace)
+	shell.Executef(t, "helm uninstall %s -n %s --ignore-not-found", c.release, c.namespace)
```

Comment:
This fails with: `Error: unknown flag: --ignore-not-found` Does it work locally for you? Maybe we just need to update helm on jenkins worker.

Reply:
Ah yes it works locally for me but maybe this was added in a later version. This is not important and can be removed. There were a couple of times where the chart failed to uninstall because it had already been deleted but I think was fixed by changing the order of cleanup operations. I can remove this.